bin/rules.d/a8-m3.rules

```diff
@@ -1,4 +1,4 @@
 # Open A8 M3 node
-SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ENV{ID_SERIAL}=="IoT-LAB_A8-M3",         ENV{ID_USB_INTERFACE_NUM}=="01",  SYMLINK+="ttyA8_M3"
+SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ENV{ID_SERIAL}=="IoT-LAB_A8-M3",         ENV{ID_USB_INTERFACE_NUM}=="01",  SYMLINK+="ttyON"
```

Comment:
I think we shouldn't change that for M3 nodes that are behind A8: this is because iotlab users may have written their own scripts using /dev/ttyA8_M3. If we change that, their scripts will be broken.

For other nodes, this is fine since the tty is only used by the gateway code.

Reply:
You're right, hadn't thought about that.